tools/job-cache/main.cpp

```diff
@@ -31,6 +32,9 @@ int main(int argc, char **argv) {
     return 1;
   }
 
+  wcl::log::subscribe(std::make_unique<wcl::log::FormatSubscriber>(std::cout.rdbuf()));
```

Comment:
A nice way to do this would be to make a FilterSubscriber type that takes a child subscriber and filters things out based on a predicate that can just be an `std::function<bool(const Entry&)>`

Reply:
This is just what we were already doing before this change.

The DaemonCache writes its logs to stdout which is redirect to `.log`

Reply:
The *client* is the issue. The client is now logging messages sent/recv to/from the server but its logging everything to the same place (stdout in this case). Also previously errors when to stderr now they go to stdout.

Reply:
The client isn't logging to stdout. 

This file is the job-cache binary which is only launched as a daemon _after_ stdout has already been replaced with a file

Reply:
ah sorry I see. Still I pointed out some cases above where we do need this.

Reply:
Ok so there were a lot of misconceptions here. I'm still not happy about errors not going to stdout but Ashley's compromise is good enough for now.